tests/e2e/tests/critical-path/memory-efficiency/memory-efficiency.e2e.ts

```diff
@@ -0,0 +1,176 @@
+
+        // Verify that sorting by Total Memory from big to small applied by default
+        await t.expect(memoryEfficiencyPage.tableRows.nth(0).textContent).contains(keySpaces[0], 'Biggest memory keyspace is not at top');
+        await t.expect(memoryEfficiencyPage.tableRows.nth(14).textContent).contains(keySpaces[2], 'Smallest memory keyspace is not at down');
+
+        await t.click(memoryEfficiencyPage.expandArrowBtn);
+        // Verify that Key Pattern with >1 keys can be expanded
+        await t.expect(memoryEfficiencyPage.expandedRow.find('button').count).eql(2, 'Expandable row has no items');
+        // Verify that user can quickly set the filters per keyspaces in the Browser/Tree View from the list of keyspaces
+        await t.click(memoryEfficiencyPage.expandedRow.find('button').nth(0));
+        // Verify filter by data type applied
+        await t.expect(await browserPage.filteringLabel.textContent).eql('Stream', 'Key type lable is not displayed in search input');
+        // Verify keyname in search input prepopulated
+        await t.expect(await browserPage.filterByPatterSearchInput.withAttribute('value', keySpaces[0]).exists).ok('Filter per key name is not applied');
+        // Verify key is displayed
+        await t.click(browserPage.browserViewButton);
+        await t.expect(await browserPage.isKeyIsDisplayedInTheList(streamKeyName)).ok('Key is not found');
+
+        // Clear filter
+        await t.click(browserPage.treeViewButton);
+        await t.click(browserPage.clearFilterButton);
+        // Change delimiter
+        await browserPage.changeDelimiterInTreeView('-');
+        // Go to Analysis Tools page
+        await t.click(myRedisDatabasePage.analysisPageButton);
+        // Create new report
+        await t.click(memoryEfficiencyPage.newReportBtn);
+        // Verify that delimiter can be changed in Tree View and applied
+        await t.expect(memoryEfficiencyPage.tableRows.count).eql(1, 'New delimiter not applied');
+        await t.expect(memoryEfficiencyPage.tableRows.nth(0).textContent).contains(keySpaces[5], 'Keyspace not displayed');
+    });
+test
+    .before(async t => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        await browserPage.addHashKey(keySpaces[4], keyTTL, hashValue);
+        await cliPage.addKeysFromCliWithDelimiter('MSET', 5);
+        await t.click(browserPage.treeViewButton);
+        // Go to Analysis Tools page
+        await t.click(myRedisDatabasePage.analysisPageButton);
+    })
+    .after(async t => {
+        await cliPage.deleteKeysFromCliWithDelimiter(5);
+        await t.click(myRedisDatabasePage.browserButton);
+        await t.click(browserPage.browserViewButton);
+        await browserPage.deleteKeyByName(keySpaces[4]);
+        await deleteStandaloneDatabaseApi(ossStandaloneConfig);
+    })('Namespaces sorting', async t => {
+    // Create new report
```

Comment:
Sorting can be covered on tests level lower (e.g. unit tests)

Reply:
Amir said that sorting is covered in elastic itself, but I decided to cover it on the high level too for sure